### PR TITLE
test(visreg): Expand to elements

### PR DESCRIPTION
**Pull request by Droogans**
### LGTMs
- [x] Dev LGTM
- [x] Design LGTM

### CODE REVIEW

File: src/elements/Links/docs/examples/link.icons.reserved.html [hunk 1]
```diff
@@ -11,6 +11,8 @@
 </div>
 
 <h4> List of Current Action Links in Encore (with their reserved icons) </h4>
+<div layout="row">
+<ul id="all-icons" flex="40">
```
Comment (CITguy) [inline src/elements/Links/docs/examples/link.icons.reserved.html]: Is there a specific need to convert to flexbox layout?

Comment (Droogans) [inline src/elements/Links/docs/examples/link.icons.reserved.html]: Not really a "need" per say, but it certainly made the screenshots come out better:

https://github.com/comeatmebro/encore-ui-screenshots/blob/52add6a4ea107a78690c2e75658808a3172c4e36/firefox/utils/visual-regression/components/rxActionMenu-midway/demo-component-links-all/0768x1024-By(css%20selector%2C%20%23all-icons).png

https://github.com/comeatmebro/encore-ui-screenshots/blob/52add6a4ea107a78690c2e75658808a3172c4e36/firefox/utils/visual-regression/components/rxActionMenu-midway/demo-component-links-all/1024x0768-By(css%20selector%2C%20%23all-icons).png

https://github.com/comeatmebro/encore-ui-screenshots/blob/52add6a4ea107a78690c2e75658808a3172c4e36/firefox/utils/visual-regression/components/rxActionMenu-midway/demo-component-links-all/1366x0768-By(css%20selector%2C%20%23all-icons).png

If I didn't do this, those screenshots would be ~4 times larger, and make it harder to trigger regressions in the future.

**Comment by glynnis** [comment]
Fine by me. LGTM

**Comment by 100stacks** [comment]
@Droogans, thanks for the PR overview.  LGTM

**Pull request merged**